packages/aft/lib/src/commands/integration_tests_command.dart

```diff
@@ -0,0 +1,179 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// ignore_for_file: avoid_dynamic_calls, inference_failure_on_untyped_parameter
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:aft/aft.dart';
+import 'package:interact/interact.dart';
+
+import '../devices.dart';
+import '../utils/start_flutter_process.dart';
+
+/// Command to run integration tests.
+class IntegrationTestCommand extends AmplifyCommand {
+  IntegrationTestCommand(this.args);
+
+  List<String> args;
+
+  @override
+  String get description =>
+      'Runs the appropriate integration test command for the given platform';
+
+  @override
+  String get name => 'integ';
```

Comment:
I'd prefer `aft test integration` and have `aft test` encapsulate all the testing functionality